src/lib/createStardustComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import createComponentInternal from './createComponent'
+import * as React from 'react'
+import * as _ from 'lodash'
+import {
+  ComponentSlotClasses,
+  ThemePrepared,
+  ComponentSlotStylesPrepared,
+  ComponentVariablesObject,
+} from '../themes/types'
+
+export interface RenderStardustResultConfig {
+  classes: ComponentSlotClasses
+  variables: ComponentVariablesObject
```

Comment:
not sure if we should expose `variables` and `styles` for the first iteration - it seems that the only bits client is interested in are _evaluated_ styles, so it is all about `classes`. The same generally applies to `theme` object as well - to me it seems to be safer not expose it if there is no need to.

So, to conclude, I would suggest to leave `classes` and maybe `rtl` only for the first run.

Reply:
I had just the classes in the beginning but was not sure whether we want this additional bits. Will leave only the classes and rtl for now. Thanks!